grpc/server/src/engine_server/mappings/state/contract_package.rs

```diff
@@ -51,11 +53,17 @@ impl TryFrom<state::ContractPackage> for ContractPackage {
     type Error = ParsingError;
     fn try_from(mut value: state::ContractPackage) -> Result<ContractPackage, Self::Error> {
         let access_uref = value.take_access_key().try_into()?;
+        let lock_status = if value.get_lock_status() {
```

Comment:
Same as above you could hide this in a special constructor _or_ construct it through `From<bool>`

Reply:
Added a `ContractPackageStatus::new` to help code simplification

Reply:
Used a special constructor